.ci/integrationTestSelector.groovy

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+import groovy.transform.Field
+
+/**
+  translate from human agent name to its app name .
+*/
+@Field Map mapAgentsApps = [
```

Comment:
idea: we start using this in many pipelines, we can move it to the shared library (other issue)